apis/v1alpha2/gateway_types.go

```diff
@@ -837,6 +838,12 @@ const (
 	// not resolve any Routes, and the "ResolvedRefs" status condition
 	// should not be raised in that case.
 	ListenerReasonInvalidRoutesRef ListenerConditionReason = "InvalidRoutesRef"
+
+	// This reason is used with the "ResolvedRefs" condition when
+	// one of the Listener's Routes has a BackendRef to an object in
+	// another namespace, where the object in the other namespace does
+	// not have a ReferencePolicy explicitly allowing the reference.
+	ListenerReasonRefNotPermitted ListenerConditionReason = "RefNotPermitted"
```

Comment:
Nit: This does not match the "ReferenceNotPermitted" name above. I think I prefer "Ref" here to match the other names.